### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by Innocent259**
In this project milestone I: 

1. Created the posts, comments, and likes forms
2. Added the basic CSS styling using Tailwind.css

### CODE REVIEW

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @like = Like.create(user: current_user, post: @post)
+    redirect_to user_post_path(@post.author, @post) if @like.save
+  end
```
Comment (DeliceLydia) [inline app/controllers/likes_controller.rb:6]: - [ ] I am not able to like a `post` please fix `create` method to help a user to be able to like a post

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Innocent259, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- I am able to use forms to create `posts` and `comments`
- Awesome README
- No linter errors


## Required Changes ♻️
_Check the comments under the review._


### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/likes_controller.rb (hunk 1 above)
Comment (Innocent259) [reply app/controllers/likes_controller.rb:6]: image

Comment (Innocent259) [reply app/controllers/likes_controller.rb:6]: Thank you @DeliceLydia I solved it!

**Review by shella12 — APPROVED** [review]
Hi @Innocent259, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Fixed create likes method ✔️ 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**